src/client/datascience/notebook/notebookEditor.ts

```diff
@@ -94,6 +94,14 @@ export class NotebookEditor implements INotebookEditor {
             })
         );
         disposables.push(model.onDidDispose(this._closed.fire.bind(this._closed, this)));
+        disposables.push(
+            commandManager.registerCommand(Commands.NativeNotebookRunAllCellsAbove, (uri) => this.runAbove(uri))
+        );
+        disposables.push(
+            commandManager.registerCommand(Commands.NativeNotebookRunCellAndAllBelow, (uri) =>
+                this.runCellAndBelow(uri)
+            )
+        );
```

Comment:
In old notebooks, the buttons are enabled only when the user selects a cell. In this case I don't know if we can do that.
We can create the context for it, but there's no event to keep it updated.